modules/telco-core-storage.adoc

```diff
@@ -0,0 +1,38 @@
+// Module included in the following assemblies:
+//
+// * telco_ref_design_specs/core/telco-core-ref-components.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="telco-core-storage_{context}"]
+= Storage
+
+Overview::
+Cloud native storage services can be provided by multiple solutions including {rh-storage} from Red Hat or third parties.
++
+{rh-storage} is a Ceph based software-defined storage solution for containers. It provides block storage, file system storage, and on-premises object storage, which can be dynamically provisioned for both persistent and non-persistent data requirements. Telco core applications require persistent storage.
++
+It is important to note that all storage data may not be encrypted in flight. To reduce risk the storage network should be isolated from other cluster networks. The storage network must not be reachable (routable) from other cluster networks, that is only nodes directly attached to the storage network can gain access to it.
+
+== {rh-storage}
+
+New in this release::
+
+Not applicable
+
+Description::
+
+{rh-storage-first} is a software-defined storage service for containers.
+For telco core clusters, storage support is provided by {rh-storage} storage services running externally to the application workload cluster.
+
+* {rh-storage} supports separation of storage traffic using secondary CNI networks.
```

Comment:
I would add this as a sentence to the previous paragraph - why is it a separate bullet?